src/scf/scf_callbacks.jl

```diff
@@ -40,28 +40,30 @@ function ScfDefaultCallback()
 
         # TODO We should really do this properly ... this is really messy
         if info.n_iter == 1
-            E_label = haskey(info.energies, "Entropy") ? "Free energy" : "Energy"
             label_magn = show_magn ? ("   Magnet", "   ------") : ("", "")
             label_damp = show_damp ? ("   α   ", "   ----") : ("", "")
             label_diag = show_diag ? ("   Diag", "   ----") : ("", "")
-            @printf "n     %-12s      Eₙ-Eₙ₋₁     ρout-ρin" E_label
+            @printf " n         Energy       log10(ΔE)   log10(Δρ)"
             println(label_magn[1], label_damp[1], label_diag[1])
-            @printf "---   ---------------   ---------   --------"
+            @printf "---   ---------------   ---------   ---------"
             println(label_magn[2], label_damp[2], label_diag[2])
         end
         E    = isnothing(info.energies) ? Inf : info.energies.total
         Δρ   = norm(info.ρout - info.ρin) * sqrt(info.basis.dvol)
         magn = sum(spin_density(info.ρout)) * info.basis.dvol
 
+        format_log(e) = @sprintf "%9.2f" log10(abs(e))
```

Comment:
Especially for the energy I am not convinced here: How do you detect that the energy increases? I think it is pretty important so see this easily the steps where the energy goes up and where down.

Reply:
For me this is very crucial. Similar to the order of magnitude of decrease, it's the first thing I check.

Reply:
what about the 1e-5.7 thingy

Reply:
It's actually super ugly, I'd rather leave it as we have now

Reply:
doesn't really have to be intuitive: you can just ignore it if you're not interested

Reply:
You would be suprised what can put people off a code. For sure if they don't understand the convergence output on a first look.

Reply:
Well the energy is there plain, and the heading is correct: it's just that there is possibly one extra character of information. Feels OK to me

Reply:
yeah sure, do it and we see what it looks :smile:.